templates/hello.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html.twig" %}
+{% block content %}
+    <div class="column1" style="margin-top: 10px;height: 50%;margin-left: 25%;width: 50%">
+        <div class="card" style="background-color: #17a2b8">
+            <form action="multiplication.php" method="post">
+                Nombre:<input type="text" name="vala"> <br><br>
+                <input type="submit" value="Calculer"><br><br>
+                {% for num in nums %}
```

Comment:
You have not passed variable nums to the twig template.

Reply:
I have already changed it